spec_finalization_checklist.md

```diff
@@ -0,0 +1,18 @@
+* The project team creates a release record as described in https://www.eclipse.org/projects/handbook/#pmi-commands-release
+* The project team creates two PRs request against the Jakarta EE Specification Committee specifications repository. The first
+PR provides everything requested in the PR template except the javadoc contents. The second PR would include the javadoc contents.
+These PRs are intended to provide the items that are required to validate the release, and provide the jakarta.ee website
+content for the specification. The github repository to create PRs against is https://github.com/jakartaee/specifications
+* The specification committee marks the PR as final when it is ready for voting.
```

Comment:
This is picky, but I believe Eclipse likes to use the term "ballot" instead of "voting".